src/components/TopBar/components/Nav.tsx

```diff
@@ -2,29 +2,31 @@ import React from "react";
 import styled, { keyframes } from "styled-components";
 import StyledRouterLink from "./StyledRouterLink";
 import StyledLink from "./StyledLink";
+import { useLocation } from "react-router-dom";
 
 interface NavProps {
   onDismiss: () => void;
   mobileMenu?: boolean;
 }
 
 const Nav: React.FC<NavProps> = ({ onDismiss, mobileMenu }) => {
+  const location = useLocation();
   return (
     <>
       { mobileMenu && (
         <StyledBackdrop onClick={onDismiss} />
       )}
       <StyledNav>
-        <StyledRouterLink target="/dashboard" label="Dashboard" mobileMenu={mobileMenu} onDismiss={onDismiss} />
-        <StyledRouterLink target="/umbrella" label="Projects" mobileMenu={mobileMenu} onDismiss={onDismiss}>
-          <StyledRouterLink target="/umbrella" label="Umbrella" mobileMenu={mobileMenu} onDismiss={onDismiss} />
+        <StyledRouterLink currentPath={location.pathname} target="/dashboard" label="Dashboard" mobileMenu={mobileMenu} onDismiss={onDismiss} />
```

Comment:
since all of these looks the same `currentPath={location.pathname}`, wouldnt it be better having it inside the component itself?